finetune/scheduler.py

```diff
@@ -108,21 +119,20 @@ def _close_oldest_model(self):
         else:
             LOGGER.info("No models cached -- cannot remove oldest model.")
 
-    def model_cache_key(self, model, key):
+    def model_cache_key(self, model, key, blob_key):
+        if blob_key:
+            return blob_key
```

Comment:
I think cache key should be analogous to model and append the nested key if provided.  This is a bit unfortunate as it complicates the in_cache() check for tables models.

Reply:
sorry not too sure i follow here.. i thought that model here will be the bytes of the model itself or None, hence needing cache_key aka blob url .

Reply:
ah so in table labeler, we load the model with key="etl" which means when we save the key originally with blob we would neeed to have something like `f"{model}_key={key}+{cache_key}"`

Reply:
The 2nd one. We need to include key in the cache key when included as this is when a file holds multiple models.